node/cli/src/command.rs

```diff
@@ -371,22 +371,22 @@ pub fn run() -> Result<()> {
 
 		#[cfg(feature = "try-runtime")]
 		Some(Subcommand::TryRuntime(cmd)) => {
```

Comment:
I have not used this. For my learning, can you explain what this does?

Reply:
This allows trying to simulate a runtime upgrade (which include storage migrations) connecting to a chain. Some more details in https://paritytech.github.io/try-runtime-cli/try_runtime/